src/EnergyPlus/BaseboardRadiator.cc

```diff
@@ -843,9 +841,14 @@ namespace BaseboardRadiator {
                             SimHWConvective(state, baseboardNum, LoadMet);
 
                             if (LoadMet > DesCoilLoad) { // if the load met is greater than design load, OK to iterate on UA
-                                // Invert the baseboard model: given the design inlet conditions and the design load,
-                                // find the design UA.
-                                General::SolveRoot(state, Acc, MaxIte, SolFla, UA, HWBaseboardUAResidual, UA0, UA1, Par2);
+                                // Invert the baseboard model: given the design inlet conditions and the design load, find the design UA.
+                                auto f = [&state, &baseboardNum, &DesCoilLoad](Real64 UA) {
```

Comment:
`baseboardNum` and `DesCoilLoad` should be passed by value, i.e., `=baseboardNum`, `=DesCoilLoad`.

Reply:
Good catch.  When I first started, I noticed that the Par array was used _after_ the call to SolveRoot sometimes, so my gut reaction was to pass by reference.  That way if some usage was doing some voodoo, at least I wouldn't cause diffs.  Looking back, it wouldn't have mattered, so yeah, value all the way (unless it's `state` of course)

Reply:
For reference, in the capture block, you don't need an `=` prefix on the variables.  If you just specify the variable name, it implies pass by value.  (`=varName` appears to be a syntax error)  I'm updating the already-converted instances and will use this pattern moving forward.

Reply:
I think I knew that. Not sure what I was doing there.